src/Components/Aspire.Microsoft.Azure.Cosmos/AspireAzureCosmosDBExtensions.cs

```diff
@@ -93,12 +93,8 @@ public static class AspireAzureCosmosDBExtensions
             });
         }
 
-        if (settings.IgnoreEmulatorCertificate && CosmosUtils.IsEmulatorConnectionString(settings.ConnectionString))
+        if (CosmosUtils.IsEmulatorConnectionString(settings.ConnectionString))
         {
-            clientOptions.HttpClientFactory = () => new HttpClient(new HttpClientHandler()
-            {
-                ServerCertificateCustomValidationCallback = HttpClientHandler.DangerousAcceptAnyServerCertificateValidator
-            });
             clientOptions.ConnectionMode = ConnectionMode.Gateway;
             clientOptions.LimitToEndpoint = true;
```

Comment:
Should these options be set when talking to the emulator? I tried commenting them out and Cosmos sample failed to work.

Reply:
My understanding is that the incoming change from the Cosmos team will do this for us. Some get to remove this code.

Reply:
Both mode and LimitToEndpoint can be removed. 

Documentation was stale and needs to refresh. Is the cause for both gateway-mode and limit
https://learn.microsoft.com/en-us/azure/cosmos-db/how-to-develop-emulator?tabs=windows%2Ccsharp&pivots=api-nosql

[ServerCertificateCustomValidationCallback](https://github.com/Azure/azure-cosmos-dotnet-v3/commit/c29abfedcbd98edf5ac49c9e58c3befc21dd9c3e) do cover for both gateway and direct modes.

Reply:
I've tried removing both and it breaks. I've tried removing them individually and it breaks.

I'm not going to remove either.